File: ruby/lib/Sikaru/import_sessions/types/create_source_artifact_request_export_policy.rb
# frozen_string_literal: true

module Sikaru
  module ImportSessions
    module Types
      module CreateSourceArtifactRequestExportPolicy
        extend Sikaru::Internal::Types::Enum

        CUSTOMER_SOURCE_EXPORTABLE = "customer_source_exportable"
        CUSTOMER_SOURCE_PRIVATE = "customer_source_private"
        REFERENCE_ONLY = "reference_only"
      end
    end
  end
end

File: swift/Sources/Schemas/CreateSourceArtifactRequestExportPolicy.swift
import Foundation

public enum CreateSourceArtifactRequestExportPolicy: String, Codable, Hashable, CaseIterable, Sendable {
    case customerSourceExportable = "customer_source_exportable"
    case customerSourcePrivate = "customer_source_private"
    case referenceOnly = "reference_only"
}